templates/data-providers/index.jsx

```diff
@@ -1,5 +1,7 @@
 import React, { useEffect, useRef, useLayoutEffect } from 'react'
 import { Button } from '@oacore/design'
+import countries from 'i18n-iso-countries'
+import englishLocale from 'i18n-iso-countries/langs/en.json'
```

Comment:
I am afraid, we will hit the performance wall soon with this. Can you use the JSON only?

Reply:
Not really this should keep the bundle as small as possible.

Reply:
How? Importing the library doesn't eliminate unused countries

Reply:
The bundle should contain only English names. And the rest of the library has like 300lines of code but I will change as we may spare those lines...